awx/main/signals.py

```
@@ -319,6 +328,37 @@ def disable_activity_stream():
+        _deferred_awx_activity_stream.active = False
+        _deferred_awx_activity_stream.pending = []
+        raise
+    else:
+        pending = _deferred_awx_activity_stream.pending
+        _deferred_awx_activity_stream.active = False
+        _deferred_awx_activity_stream.pending = []
+
+        if pending:
+            entries = [entry for entry, _ in pending]
+            ActivityStream.objects.bulk_create(entries)
+            for entry, on_commit_cb in pending:
```

Ruff (0.15.21)

[warning] Loop control variable `entry` not used within loop body

Rename unused `entry` to `_entry`

(B007)